### DISCUSSION TITLE
Project Reunion is now called the Windows App SDK! 🎉

### DISCUSSION THREAD

**Discussion by tfennel** [microsoft/WindowsAppSDK#966] [Announcements]
Today, we just announced the final name for Project Reunion - the Windows App SDK! We can't wait to see the amazing apps you build with the Windows App SDK! 😍

## What this means

Many of our assets, like the docs, GitHub page, and more, have been renamed. We will be using the name Windows App SDK from this point onwards and you should too! If you ask questions on StackOverflow, use the tag `windows-app-sdk`.

## Why is the 0.8 NuGet and VSIX still called Project Reunion?

We weren't able to fully rename everything yet. Some assets, like the NuGet packages and VSIX, are still using the codename. **The 0.8 NuGet/VSIX will remain named Project Reunion**. We will switch these assets to the new name in a future release. You should still refer to 0.8 as "Windows App SDK 0.8" rather than "Project Reunion 0.8", unless specifically referring to the NuGet or VSIX.

## What happens to links?

All links that had Project Reunion in the URL should auto-redirect. For example, links to the GitHub will automatically redirect. Even if you've previously cloned the repository, pulling and pushing will still continue to work. Links to the docs pages should also redirect. If you notice any broken links, please let us know by filing an issue!

**Comment by pjmagee** [comment]
Will the NuGet package be renamed to WindowsAppSDK before the 1.0 release?
